gruntfile.js

```diff
@@ -157,4 +162,5 @@ module.exports = function(grunt) {
     grunt.registerTask('default', ['install', 'versionupdater', 'compress']);
     grunt.registerTask('version', ['versionupdater']);
     grunt.registerTask('zip', ['compress']);
+    grunt.registerTask('client:lint', 'Check JS syntax', ['eslint']);
```

Comment:
Apparently, the convention is `task:target`, that means `lint:client`.

https://gruntjs.com/configuring-tasks#task-configuration-and-targets